RecoHGCal/TICL/src/classes.h

```diff
@@ -0,0 +1,11 @@
+#include <vector>
+#include "FWCore/Common/interface/EventBase.h"
+#include "RecoHGCal/TICL/interface/Trackster.h"
+
+namespace RecoHGCal_TICL {
```

Comment:
the namespace_dictionary detail is no longer necessary. Only the includes are needed.